open-api/rest-catalog-open-api.yaml

```diff
@@ -1785,6 +1907,24 @@ components:
         - "is-nan"
         - "not-nan"
 
+    TrueExpression:
```

Comment:
Was there any reason the true and false expressions were not added in the spec originally? @nastra

Reply:
there was no particular reason, most likely it was just missed